CRM/Core/BAO/CustomValueTable.php

```diff
@@ -536,16 +536,17 @@ public static function &getEntityValues($entityID, $entityType = NULL, $fieldIDs
    * @return array
    */
   public static function setValues(&$params) {
+    // For legacy reasons, accept this param in either format
+    if (empty($params['entityID']) && !empty($params['entity_id'])) {
+      $params['entityID'] = $params['entity_id'];
+    }
```

Comment:
The param is actually `entity_id` so why this function ever expected a different format is a mystery. For legacy reasons we'll keep accepting `entityID` but at least now it recognizes the correct input.